Bio/SeqRecord.py

```diff
@@ -667,7 +667,7 @@ def __repr__(self):
         ...                 id="NP_418483.1", name="b4059",
         ...                 description="ssDNA-binding protein",
         ...                 dbxrefs=["ASAP:13298", "GI:16131885", "GeneID:948570"])
-        >>> print(repr(rec))
+        >>> rec
```

Comment:
Is it worth leaving this one since it is in the ``__repr__`` docstring, otherwise I think the text above needs a slight rewording.